community/oath-toolkit/depends

```diff
@@ -0,0 +1,2 @@
+pkgconf make
```

Comment:
This is an implicit dependency and can be removed (same as `make`, `gcc` for reference).